local-config.yml

```diff
@@ -20,7 +20,7 @@ routes:
   - name: q
     value: bench-routes
 - method: GET
-  url: https://www.bing.com
+  url: https://www.facebook.com
```

Comment:
I think this was failing in some platforms. Anyways, if we need an extra URL for the test, we can create a new route, rather than changing the existing test URL.

Reply:
Yaa sure.. I will update it accordingly.